priv/repo/migrations/20200227152752_create_events.exs

```diff
@@ -0,0 +1,34 @@
+defmodule Erlef.Data.Repo.Migrations.CreateEvents do
+  use Ecto.Migration
+
```

Comment:
@ghislaineguerin @nurugger07 Let's suss out what is required and what is not and make it so here.